common/app/conf/GoogleAuth.scala

```diff
@@ -0,0 +1,52 @@
+package conf
+
+import conf.Configuration.OAuthCredentials
+import com.amazonaws.auth.AWSCredentialsProviderChain
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder
+import com.gu.googleauth.{AntiForgeryChecker, GoogleAuthConfig}
+import com.gu.play.secretrotation.{SnapshotProvider, TransitionTiming}
+import play.api.http.HttpConfiguration
+
+import java.time.Duration.{ofHours, ofMinutes}
+
+case class GoogleAuth(
+    currentHost: Option[String],
+    httpConfiguration: HttpConfiguration,
+    oauthCredentials: Option[OAuthCredentials],
+) {
+  private val securityCredentialsProvider = new AWSCredentialsProviderChain(
+    Configuration.aws.mandatoryCredentials,
+  )
+  private val ssmClient = AWSSimpleSystemsManagementClientBuilder
+    .standard()
+    .withCredentials(securityCredentialsProvider)
+    .withRegion(Regions.EU_WEST_1)
+    .build()
+
+  val secretStateSupplier: SnapshotProvider = {
+    import com.gu.play.secretrotation.aws.parameterstore
+
+    new parameterstore.SecretSupplier(
+      TransitionTiming(usageDelay = ofMinutes(3), overlapDuration = ofHours(2)),
+      "/frontend/PlayAppSecret",
```

Comment:
you may want to vary this by stage at least? i.e. `/frontend/CODE/PlayAppSecret`.

I'd also consider sticking closer to the conventions in https://github.com/guardian/frontend/blob/b1b94271163668a38887df8434829be23852613a/common/app/common/configuration.scala. By that I mean probably `playAppSecret`, possibly picking up namespace if appropriate e.g. `oauth.playAppSecret` (haven't looked fully at what's there) and probably moving some of this inside that file.

Reply:
Thanks! Modified this to be different for each app (admin and preview) and each stage.

Reply:
Created an issue for the rotation part: https://github.com/guardian/frontend/issues/24816